### PR TITLE
Use JCB for aerosol DA

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
What the title says.

Depends on:
https://github.com/NOAA-EMC/jcb-gdas/pull/33
https://github.com/NOAA-EMC/jcb-algorithms/pull/5

### CODE REVIEW

**Comment by CoryMartin-NOAA** [comment]
Will need hashes potentially updated after merge of the dependent PRs

**Review by DavidNew-NOAA — DISMISSED** [review]

**Comment by CoryMartin-NOAA** [comment]
DO NOT MERGE yet

Edit. all good now

**Review by DavidNew-NOAA — APPROVED** [review]
Looks solid

**Pull request merged**
